src/commands/contract/interfaces.ts

```diff
@@ -46,30 +46,28 @@ export function generateActionInterface(
             types = abiVariant.types
 
             const variantTypeNodes = types.map((type) =>
-                ts.factory.createTypeReferenceNode(parseType(findExternalType(type, 'Types.', abi)))
+                ts.factory.createTypeReferenceNode(parseType(findExternalType(type, 'Base.', abi)))
```

Comment:
Using `Base` here doesn't really make sense to me 🤔 How about using `_Types` instead?